### PR TITLE
Cosmetics

### PR DESCRIPTION
**Pull request by tnowotny**
Just some name changes and removing of unnecessary parameters in the Mbody example. Note: I was not able to test as I somehow have broken my brian2genn setup. However, unless I introduced typos all changes should be purely cosmetic.

### CODE REVIEW

File: Mbody_example.py [hunk 1]
```diff
@@ -114,59 +101,59 @@
 spike_times = spike_times.repeat([len(p) for p in sorted_variants])
 spike_indices = np.concatenate(sorted_variants)
 
-PN = SpikeGeneratorGroup(N_AL, spike_indices, spike_times)
+PN = SpikeGeneratorGroup(N_PN, spike_indices, spike_times)
 
 # iKC of the mushroom body
-I_syn = '''I_syn = g_PN_iKC*(V - E_e): amp
-           dg_PN_iKC/dt = -g_PN_iKC/tau_PN_iKC : siemens'''
+I_syn = '''g_PNiKC : siemens
```
Comment (mstimberg) [inline Mbody_example.py]: I think removing the decay of the synaptic conductance was probably not intended here?

**Review by mstimberg — CHANGES_REQUESTED** [review]

File: Mbody_example.py (hunk 1 above)
Comment (tnowotny) [reply Mbody_example.py]: Indeed not - I have no clue how this happened as I was only doing query-replace in emacs for the variable names ... a mystery. Anyway - I think I have reverted to what it should be (please check).

**Comment by mstimberg** [comment]
Did you maybe base your file on a different version of the example, one where you tried out summed variables? I only spotted the difference in the `I_syn` equations, but the whole definition of `PN_iKC` changed to using a summed variable (which is not compatible with the current/old definition of `I_syn`).

**Comment by tnowotny** [comment]
that would be a possible explanation. It can sometimes get a bit confusing if one is not extremely careful. Do you think we have it now sorted or should I start over?

**Comment by mstimberg** [comment]
It does not currently work, since the definition of `PN_iKC` has changed the synapse model and is currently using a summed variable. Maybe just reverting its definition (except for the renames) is enough. I'll try that and see whether the code runs (and gives the same results as before).

**Comment by tnowotny** [comment]
If you could try that, that would be much appreciated ... this must have been a non-committed edit I had made when doing experiments with summed vars and then got transferred into the new branch when I made it ...

**Comment by mstimberg** [comment]
Um, I think I fixed the synapse model, but when I run the simulation with scaling factor 5, I get slightly different firing rates for the `eKC` neurons than before. In the master branch, running a few simulations I get between 9.3 and 10.4 spikes/s, whereas in the cosmetics branch I get between 13.1 and 13.8... I do not quite see where the difference comes from, though, need to look into this more.

**Comment by tnowotny** [comment]
very strange ... I really can't see any more differences. The only thing I changed consciously that is not literally a name change was that I inverted g_scaling to get "k" and then everything is scaled x*k instead of x/g_scaling. Oh dear - is that doing an integer division?!?

**Comment by mstimberg** [comment]
>  Oh dear - is that doing an integer division?!?

Um, in Brian code we do take care of this, but I'm not 100% sure about Brian2GeNN... The `2500/N_iKCeKC` should be fine though, I compared the examples under Python 3 where this is a floating point division. But we should put a future import in there so that it does the same thing under Python 2.

File: Mbody_example.py [hunk 2]
```diff
@@ -29,47 +19,44 @@
...
 VT = -63*mV
-# Those two constants are dummy constants, only used when populations only have
-# either inhibitory or excitatory inputs
-E_e = 0*mV
-E_i = -92*mV
-# Actual constants used for synapses
-NKCKC= N_MB
-if NKCKC > 10000:
-    NKCKC = 10000
-g_scaling = NKCKC/2500
```
Comment (tnowotny) [inline Mbody_example.py:52]: The problem would be whether this does an integer division (would by a pure Python problem).

Comment (tnowotny) [reply Mbody_example.py:52]: ... but if you tested in Python 3 and it's not an integer division in Python 3 it should be fine...

Comment (tnowotny) [reply Mbody_example.py:52]: But to be 100% sure, could you revert to the g_scaling variant and test?

**Comment by mstimberg** [comment]
Ouch, I finally found the bug:
```Python
g_scaling = NKCKC/2500
if g_scaling < 1:
    g_scaling= 1
```
was converted into
```Python
k = 2500/N_iKCeKC
if k < 1:
    k = 1
```
The if prevented any actual scaling of the weights :laughing:

**Comment by tnowotny** [comment]
Oh dear ... my brain was clearly not working. Many apologies for the mess. That wasn't necessary :-(

**Comment by mstimberg** [comment]
No worries. It seems to work now correctly, I'll do a few more tests and then go ahead with the merge.

**Comment by mstimberg** [comment]
Ok, all looks good to me, results seem to be the same as previously both with C++ standalone and with GeNN

**Review by mstimberg — APPROVED** [review]

**Pull request merged**